File: AtCoder Beginner Contest 192/B - uNrEaDaBlE sTrInG.py
# https://atcoder.jp/contests/abc192/tasks/abc192_b

import sys

s = list(input())

n = len(s)

# print(s)

# print(s[0].islower())


for i in range(n):
    # print(s[i])
    if(i % 2 == 0):
        if(s[i].islower() == True):
            pass
        else:
            print("No")
            sys.exit()
    else:
        if(s[i].isupper() == True):
            pass
        else:
            print("No")
            sys.exit()

print("Yes")

File: AtCoder Beginner Contest 192/B - uNrEaDaBlE sTrInG.cpp
// https://atcoder.jp/contests/abc192/tasks/abc192_b

#include<bits/stdc++.h>
using namespace std;

int main(){
  string S;
  cin >> S;

  //bool b = 1;
  for(int i = 0; i < S.length(); i++){
    if(i%2 == 0){
      if(isupper(S[i])){
        cout << "No" << endl;
        //b = 0;
        return 0;
      }
    }else{
      if(islower(S[i])){
        cout << "No" << endl;
        //b = 0;
        return 0;
      }
    }
  }
  //if(b){
    cout << "Yes" << endl;
  //}
  return 0;
}
